test/simulator/misc/silvally.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('./../../common');
+
+const unimportantPokemon = {species: 'magikarp', moves: ['splash']};
+
+describe(`[Hackmons] Silvally`, function () {
+	it(`in untyped forme should change its type to match the memory held`, function () {
+		const battle = common.gen(4).createBattle([
```

Comment:
Maybe I'm missing something, but is there a reason all these tests are being done with Gen 4 mechanics?

Reply:
Oh, I didn't notice. I just copied this out of the arceus.js test file.